doc/content/getting-started/platform-guide/_index.md

```diff
@@ -5,27 +5,87 @@ weight: -1
+
+Once the camera is in focus, console would be able to successfuly scan the QR code and extract the required device credentials. Please hit **Apply** to continue the registration process.
+
+{{< figure src="gnse_claim_3.png" alt="QR code scanned" >}}
+
+{{% gnse %}} is already included in the [device repository](https://www.thethingsnetwork.org/device-repository/), so you just input the device brand, model, region profile and the frequency plan for your location (i.e. The Things Industries -> Generic Node (SE), EU_863_870, SF9 for RX2 - recommended ) and simply hit **Register end device**.
+
+{{< figure src="gnse_claim_4.png" alt="Registering end device" >}}
+
+Congratulations! You have successfully registered your {{% gnse %}} device. You may proceed to ***Step 5***.
+
+{{< figure src="gnse_claim_7.png" alt="Device registered" >}}
+
+
+## Step 5: Powering up your device
+
+After claiming, you can power up the device using two standard AA batteries or with a small 3.7V LiPo/Li-ion battery with a standard JST PH 2.0 connector.
+
+If the device powers up correctly, you will see the green LED briefly lighting up indicating successful boot-up. Pressing the front user button also triggers a sensor sampling followed by an uplink.
+
+## Step 6: Joining the network and seeing the first data packets
+
+The device will attempt to join LoRaWAN network using OTAA. Please make sure there's a TTN/TTS LoRaWAN Gateway coverage in your area. 
+
+When the device joins successfully you will see some verbose activity and raw payload in the ***Live Data*** tab of the device console. If you are not able to understand the raw bytes, don't panic!, it's because we still need to modify the payload formatter. Please copy-paste the custom Javascript formatter code below to the ***Payload Formatters -> Uplink*** section of your application. 
 
-For example, if you are planning to use the {{% gnse %}}, you need:
+{{< figure src="gnse_claim_8.png" alt="Payload Formatter" >}}
 
-1. {{% gnse %}} board
-2. {{% gnse %}} application
 
-## Get a device
+```javascript
+function decodeUplink(input) {
+  var data = {};
+  data.batt_volt = (input.bytes[0]/10);
+  data.temperature = (((input.bytes[1] << 8) + input.bytes[2]) - 500)/10;
+  data.humidity = ((input.bytes[3] << 8) + input.bytes[4])/10;
+  data.button = input.bytes[5];
 
-{{% gnse %}} is currently in the production phase. Order one at [genericnode.com](https://www.genericnode.com/)!
+  return {
+    data: data,
+  };
+}
+```
 
-## Explore your node's features
+If everything goes well you should see data packets in correctly formatted form. Well done!
+By default, the devices are configured to transmit every 10 minutes. You're free to change this by sending hexadecimal values in minutes as a downlink message to the device under ***Messaging -> Downlink*** on FPort 1. 
```

Comment:
```suggestion

By default, devices are configured to transmit every 10 minutes. You're free to change this by sending hexadecimal values in minutes as a downlink message to the device under **Messaging -> Downlink** on FPort 1. 
```

Reply:
Check if these arrows -> render nicely, because I don't think they are. I think you have to use a markdown code for arrow